packages/origin-247-claim/src/utils/aggregate.ts

```diff
@@ -89,26 +89,29 @@ export const aggregate = ({
         end,
         timezoneOffset
     });
-    const intervals = buildAggregateFrames(offsettedCommand);
+    const dataset = start
+        ? offsettedCommand.data.filter((d) => d.time > offsettedCommand.start!)
+        : offsettedCommand.data;
+
+    const groupedByInterval = new Map<string, BigNumber[]>();
```

Comment:
Use `Record<string, BigNumber[]>` instead of map, it will be simpler, the code will look a bit better, and much faster. That's easy change.

Thanks